### PR TITLE
release-notes en

### PR DESCRIPTION
**Pull request by Hearen**
#51

### CODE REVIEW

File: site/src/site/sphinx/en/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+* [improvement]  [#96] [`sc`](sc.md) and more commands are supporting format as `com/taobao/xxx/TestClass`
+* [bug] [#124] fix the negative values of [`trace`](trace.md)
+* [improvement]  [#128] the output of [`tt`](tt.md) will auto-expand now
+* [bug] [#130] providing more meaningful error messages when port conflicts
+* [bug] [#98] fix Arthas starting issue: when updating/downloading failed, Arthas will fail to start
+* [bug] [#139] fix agent attaching fails under some scenarios issues
+* [improvement]  [#156] delay `jd-core-java` initialization to avoid Arthas starting failure
+* [bug] avoid thread names duplicate issue
+* [improvement]  [#150] filtering by total time cost in [`trace`](trace.md) 
+* [bug] fix [`sc`](sc.md) `NPE issue when searching `SystemClassloader`
```
Comment (ralf0131) [inline site/src/site/sphinx/en/release-notes.md]: The format looks malformed starting from here.

Comment (ralf0131) [reply site/src/site/sphinx/en/release-notes.md]: `NPE issue when searching`  I think a '`' missing here.

**Comment by Hearen** [comment]
@ralf0131 Yeah...thank you 👍 Just too many links and backticks ``` there. Fixed and `push -f`ed.

File: site/src/site/sphinx/en/release-notes.md (hunk 1 above)
Comment (Hearen) [reply site/src/site/sphinx/en/release-notes.md]: Many thanks, 👍 too many links and `` ` `` there. Fixed by `push -f`ed.

**Comment by ralf0131** [comment]
LGTM now, thanks!

**Pull request merged**